client/app/pages/dashboards/edit-dashboard-dialog.html

```diff
@@ -5,7 +5,7 @@ <h4 class="modal-title">Edit: {{$ctrl.dashboard.name}}</h4>
 <div class="modal-body">
   <p>
     <form name="myForm">
-      <input name="dashboardName" type="text" class="form-control" placeholder="Dashboard Name" ng-model="$ctrl.dashboard.name" required>
+      <input name="dashboardName" type="text" class="form-control" placeholder="Dashboard Name" ng-model="$ctrl.dashboard.name" ng-keyup="$event.keyCode == 13 && $ctrl.saveDashboard()" required>
```

Comment:
This seems a bit hacky - I'd rather it call a function that does this check and also comment that keycode 13 is 'enter' for example. Good catch though!